adafruit_mcp9600.py

```diff
@@ -35,6 +35,11 @@
 from adafruit_register.i2c_bits import RWBits, ROBits
 from adafruit_register.i2c_bit import RWBit, ROBit
 
+try:
+    from busio import I2C
```

Comment:
Lets add an "unused" typing import here as the first import so that the exception will get raised on the microcontroller when it runs. That will prevent importing ITC when it's not needed at runtime on the microcontroller. There is an example in this library: 

https://github.com/adafruit/Adafruit_CircuitPython_NTP/blob/acc7d3608aa33e4740f9ac28eb4d5230b5eb86d7/adafruit_ntp.py#L26-L27